serverless/packages/functions/src/dbutils/put_trainspace.ts

```diff
@@ -0,0 +1,24 @@
+import { TrainStatus } from '../trainspace/constants';
+import { PutCommandInput } from "@aws-sdk/lib-dynamodb";
+
+export function create_trainspace(trainspaceId: string, user_id: string, dataSource: string, dataset_data: object, trainspaceName: string, parameters_data: object, reviewData: string) : PutCommandInput | null
+{
+    let output: PutCommandInput = 

```

Comment:
In the future, we plan to add other trainspace types such as audio, cnns, neural net, etc that may make this function to cumbersome to generalize